easybuild/easyconfigs/t/Trinity/Trinity-2.0.6-goolf-1.4.10.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'Trinity'
+version = '2.0.6'
+
+homepage = 'http://trinityrnaseq.github.io/'
+description = """ Trinity assembles transcript sequences from Illumina RNA-Seq data. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/archive/']
+sources = ['v%(version)s.tar.gz']
+
+unpack_options = '--strip-components=1'
+
+java = 'Java'
+javaver = '1.7.0_21'
```

Comment:
I would have asked for 1.8.x, if I didn't have doubts about bioinfo
packages having issues with 1.8.x
(can someone make a statement of compatibility here? that would resolve the
tie)

On 3 June 2015 at 20:14, Kenneth Hoste <notifications@github.com> wrote:

> In easybuild/easyconfigs/t/Trinity/Trinity-2.0.6-goolf-1.4.10.eb
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1663#discussion_r31659107>
> :
>
> > +
> > +name = 'Trinity'
> > +version = '2.0.6'
> > +
> > +homepage = 'http://trinityrnaseq.github.io/'
> > +description = """ Trinity assembles transcript sequences from Illumina RNA-Seq data. """
> > +
> > +toolchain = {'name': 'goolf', 'version': '1.4.10'}
> > +
> > +source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/archive/']
> > +sources = ['v%(version)s.tar.gz']
> > +
> > +unpack_options = '--strip-components=1'
> > +
> > +java = 'Java'
> > +javaver = '1.7.0_21'
>
> +1, for the latest version, this Java dep is *way* too far behind
>
> 1.7.0_80, at least, maybe latest 1.8.x
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1663/files#r31659107>
> .
>



-- 


echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum